l10n_eu_oss/wizard/l10n_eu_oss_wizard.py

```diff
@@ -90,6 +90,9 @@ def _default_todo_country_ids(self):
         comodel_name="account.tax", string="Second Super Reduced Tax"
     )
 
+    def _prepare_tax_group_vals(self, rate):
+        return {"name": _("OSS VAT {}{}".format(rate, "%"))}
```

Comment:
Use old replacing system (`%s`) and don't include in the translatable part the replacing itself:
```suggestion
        return {"name": _("OSS VAT %s%%") % rate}
```